cmd/eventlistenersink/main.go

```diff
@@ -83,7 +86,26 @@ func main() {
 	profilingServer := profiling.NewServer(profilingHandler)
 	metrics.MemStatsOrDie(ctx)
 
-	sharedmain.WatchObservabilityConfigOrDie(ctx, configMapWatcher, profilingHandler, logger, EventListenerLogKey)
+	exp := metrics.ExporterOptions{
+		Domain:         "",
```

Comment:
I don't know a ton about metrics but from reading the docs for ExporterOptions, it says that this field "Must be present". Is "" as valid value?

Reply:
You are right 
if its empty by default it takes value from here https://github.com/tektoncd/triggers/blob/main/vendor/knative.dev/pkg/metrics/exporter.go#L123-L125 which internally takes value from env `METRICS_DOMAIN`